sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -1020,6 +1020,18 @@ private case class GpuOrcFileFilterHandler(
             setMapping(id)
             updateMapping(prefixNew, children(i))
           }
+        } else if(schema.getCategory == TypeDescription.Category.MAP) {
+          val children = schema.getChildren.asScala
+          for (i <- 0 until children.size) {
```

Comment:
NIT:  A more Scala way.
```suggestion
          children.indices.foreach { i =>
```

Reply:
good tip, fixed